llama_stack/providers/remote/vector_io/milvus/milvus.py

```diff
@@ -102,9 +150,10 @@ async def query_vector(self, embedding: NDArray, k: int, score_threshold: float)
             self.client.search,
             collection_name=self.collection_name,
             data=[embedding],
+            anns_field="vector",
             limit=k,
             output_fields=["*"],
-            search_params={"params": {"radius": score_threshold}},
+            search_params={"metric_type": "COSINE", "params": {"score_threshold": score_threshold}},
```

Comment:
we should probably just pass params here and just have some special handling for the `score_threshold` params key.

Reply:
I have left the metric_type and just passed on score_threshold for now to maintain similarity with how vector search for implemented previously. Passing params directly, would require us to introduce an API change which is what I did in https://github.com/meta-llama/llama-stack/pull/2465, and there were preferences in passing typed params rather than generic maps leaving validation to providers. Leaving it for follow up, as that would require further discussion.